### PR TITLE
Update Elixir on GKE tutorial to cover Phoenix 1.4 and Cloud SQL

### PR DESCRIPTION
**Pull request by dazuma**
This is a long-needed update to the Elixir Phoenix on GKE tutorial.

It updates the tutorial to cover recent updates to the Phoenix framework (version 1.4) and the Distillery build tool (version 2.0). More significantly, it now includes material on how to connect to a Postgres database running in Cloud SQL, a very common request from readers.

This supersedes #414 by including a custom `.gcloudignore` that does not omit `prod.secret.exs`.

See also the corresponding update to the Elixir Phoenix on GAE tutorial in #605

### CODE REVIEW

File: tutorials/elixir-phoenix-on-kubernetes-google-container-engine/index.md [hunk 1]
```diff
@@ -3,35 +3,37 @@ title: Run an Elixir Phoenix app in containers using Google Kubernetes Engine
...
-earlier known as Google Kubernetes Engine, is based on the popular open-source
-[Kubernetes](https://kubernetes.io/) system, and leverages Google's deep
-expertise with container-based deployments.
+Google's hosting solution for containerized applications. Kubernetes Engine is
+based on the popular open-source [Kubernetes](https://kubernetes.io/) system,
+and leverages Google's deep expertise with container-based deployments.
 
 You will create a new Phoenix application, and then you will learn how to:
 
+*   Connect your app to a database running in Google Cloud SQL
```
Comment (ccarpentiere) [inline tutorials/elixir-phoenix-on-kubernetes-google-container-engine/index.md]: Google Cloud SQL -> Cloud SQL

File: tutorials/elixir-phoenix-on-kubernetes-google-container-engine/index.md [hunk 2]
```diff
@@ -3,35 +3,37 @@ title: Run an Elixir Phoenix app in containers using Google Kubernetes Engine
...
-This tutorial requires Elixir 1.4 and Phoenix 1.3 or later. It assumes you are
-already familiar with basic Phoenix web development. For simplicity, the
-tutorial app does not use Ecto or connect to a SQL database, but you can extend
-it to connect to Google Cloud SQL or any other database service.
+This tutorial requires Elixir 1.5 and Phoenix 1.4 or later. It assumes you are
+already familiar with basic Phoenix web development. It also requires the
+PostgreSQL database to be installed on your local development workstation.
+
+This tutorial was updated in Dec 2018 to cover Phoenix 1.4, Distillery 2.0, and
+connecting Ecto to a Google Cloud SQL database.
```
Comment (ccarpentiere) [inline tutorials/elixir-phoenix-on-kubernetes-google-container-engine/index.md]: Google Cloud SQL -> Cloud SQL

File: tutorials/elixir-phoenix-on-kubernetes-google-container-engine/index.md [hunk 3]
```diff
@@ -111,6 +141,170 @@ you already have an app to deploy, you can use it instead.
...
+1.  Visit [http://localhost:4000](http://localhost:4000) to verify that your
+    new code is running. You can log into your database and add new rows, and
+    reload the page to verify that the count has changed.
+
+For more information on using Ecto to access a SQL database, see the
+[Phoenix Ecto guide](https://hexdocs.pm/phoenix/ecto.html).
+
+## Create a production database in Cloud SQL
+
+In this section, you will create your "production" database using Google Cloud
```
Comment (ccarpentiere) [inline tutorials/elixir-phoenix-on-kubernetes-google-container-engine/index.md]: Google Cloud SQL -> Cloud SQL

Same with all subsequent references...

File: tutorials/elixir-phoenix-on-kubernetes-google-container-engine/index.md [hunk 4]
```diff
@@ -111,6 +141,170 @@ you already have an app to deploy, you can use it instead.
...
+          --instance=hellodb --prompt-for-password
+
+    When prompted, enter a password for the database.
+
+### Connect to your Cloud SQL instance
+
+In this section you will learn how to connect to your Cloud SQL instance from
+your local workstation. Generally, you will not need to do this often, but it
+is useful for the initial creation and migration of your database, as well as
+for _ad hoc_ database connection for maintenance.
```
Comment (ccarpentiere) [inline tutorials/elixir-phoenix-on-kubernetes-google-container-engine/index.md]: for _ad hoc_ database connection for maintenance  -> for creating _ad hoc_ database connections for maintenance

File: tutorials/elixir-phoenix-on-kubernetes-google-container-engine/index.md [hunk 5]
```diff
@@ -111,6 +141,170 @@ you already have an app to deploy, you can use it instead.
...
+
+By default, Google Cloud SQL instances are secured: to connect using the
+standard `psql` tool, you are required to whitelist your IP address. This
```
Comment (ccarpentiere) [inline tutorials/elixir-phoenix-on-kubernetes-google-container-engine/index.md]: you are required to -> you must

**Comment by dazuma** [comment]
Made the requested changes, and merged latest master in.

**Comment by dazuma** [comment]
Removed the angle brackets since they aren't strictly necessary.

**Comment by ToddKopriva** [comment]
Thanks, Daniel. I'll do the production pass today.

**Pull request merged**